docs/manage/fleet/_index.md

```diff
@@ -117,3 +119,14 @@ If you are collaborating on a robot and controlling it using the [**Control** ta
 ### Deployment
 
 You and your collaborators can deploy [control logic](/program/apis/), [modular resources](/modular-resources/), sidecar [processes](../configuration/#processes), or [machine learning models](/services/ml/) to your fleet of robots without manually copying files by uploading it to Viam's cloud and deploying it to your fleet.
+
+## The Viam mobile app
+
+{{<gif webm_src="/manage/mobile-app-octagon.webm" mp4_src="/manage/mobile-app-octagon.mp4" alt="GIF of red button being pressed and cannon of confetti bot spraying confetti" class="alignright" max-width="200px">}}
+
+<br>
```

Comment:
I added this break because I think it makes the tall narrow vid next to the little bit of text look slightly less awkward